README.markdown

```diff
@@ -2621,16 +2621,45 @@ But do not forget to comment this line out before publishing your site to the wo
 If you are using the [official pre-built packages](http://openresty.org/en/linux-packages.html) for [OpenResty](https://openresty.org/)
 1.11.2.1 or later, then everything should work out of the box.
 
-If you are using OpenSSL libraries not provided by [OpenResty](https://openresty.org),
-then you need to apply the following patch for OpenSSL 1.0.2h or later:
+If you are using OpenSSL libraries not provided by
+[OpenResty](https://openresty.org), then you need to apply the
+`sess_set_get_gb_yield` patch for OpenSSL 1.0.2 up to (and
+including) 1.1.0.
+
+For OpenSSL 1.0.2h and later:
 
 <https://github.com/openresty/openresty/blob/master/patches/openssl-1.0.2h-sess_set_get_cb_yield.patch>
 
-If you are not using the NGINX core shipped with [OpenResty](https://openresty.org) 1.11.2.1 or later, then you need to
-apply the following patch to the standard NGINX core 1.11.2 or later:
+For OpenSSL 1.1.0c:
+
+<https://github.com/openresty/openresty/blob/master/patches/openssl-1.1.0c-sess_set_get_cb_yield.patch>
+
+For OpenSSL 1.1.0d and later:
+
+<https://github.com/openresty/openresty/blob/master/patches/openssl-1.1.0d-sess_set_get_cb_yield.patch>
+
+No patch is needed for OpenSSL 1.1.1 and later.
+
+Similarly, if you are not using the NGINX core shipped with
+[OpenResty](https://openresty.org) 1.11.2.1 or later, then you need to apply
+the following patch to the standard NGINX core 1.11.2:
 
 <http://openresty.org/download/nginx-1.11.2-nonblocking_ssl_handshake_hooks.patch>
 
+For the 1.13.6 NGINX core, apply the following patches:
+
+<https://github.com/openresty/openresty/blob/master/patches/nginx-1.13.6-ssl_cert_cb_yield.patch>
+<https://github.com/openresty/openresty/blob/master/patches/nginx-1.13.6-ssl_pending_session.patch>
+
+For the 1.15.8 NGINX core:
+
+<https://github.com/openresty/openresty/blob/master/patches/nginx-1.15.8-ssl_cert_cb_yield.patch>
+<https://github.com/openresty/openresty/blob/master/patches/nginx-1.15.8-ssl_pending_session.patch>
+
+For subsequent NGINX cores, the patches will be updated and their name will
+contain the supported core version (e.g.
+`nginx-<major.minor.patch>-*.patch`).
+
```

Comment:
Let's just use the openresty.org links here:

https://openresty.org/en/nginx-ssl-patches.html

https://openresty.org/en/openssl-patches.html

It's too verbose to put them here in this lengthy README.

Reply:
@agentzh Updated and now much more concise with better wording. We also take the opportunity to inform users of our OpenSSL packages (https://openresty.org/en/linux-packages.html):

> If you are not using one of the OpenSSL packages provided by OpenResty, you will need to apply patches to OpenSSL 1.0.2, up to (and including) 1.1.0:
>
> https://openresty.org/en/openssl-patches.html
>
> Similarly, if you are not using the NGINX core shipped with OpenResty 1.11.2.1 or later, you will need to apply patches to the standard NGINX core:
>
> https://openresty.org/en/nginx-ssl-patches.html